app/services/local_interfaces/destroyer.rb

```diff
@@ -2,12 +2,26 @@ module LocalInterfaces
   class Destroyer
     include Serviceable
 
+    def call(environment)
+      @environment = environment
+      @service     = "#{environment.app_id}"
+      uninstall(@service) if exists?(@service)
+    end
+
+    private
+
+    attr_reader :environment
```

Comment:
Usually I have seen `attr_reader` at the top of the class, i.e. not in `private`. BUT, if you did this intentionally to keep it private, good on you and disregard!